form-runner/jvm/src/main/resources/apps/fr/components/view.xsl

```diff
@@ -499,21 +499,21 @@
                       <xh:span class="navbar-toggler-icon"></xh:span>
                     </xh:button>
                     <xh:div class="collapse navbar-collapse" id="navbarSupportedContent">
-                        <xh:ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <xh:ul class="navbar-nav ms-auto mb-2 mb-lg-0">
```

Comment:
Add `align-items-center` to center the content vertically.

Reply:
This approach will result in the navbar items cannot been aligned to the left after the navbar collasped.
I added `d-flex align-items-center` in the <li> tag (see below line 513 and 516) instead.